charts/gardener-extension-admission-alicloud/charts/application/templates/validatingwebhook-validator.yaml

```diff
@@ -17,7 +17,11 @@ webhooks:
     - cloudprofiles
     - shoots
   failurePolicy: Fail
-  objectSelector: {}
+  objectSelector:
+    {{- if .Values.global.webhookConfig.useObjectSelector }}
+    matchLabels:
+      provider.extensions.gardener.cloud/alicloud: "true"
```

Comment:
I don't know how the filtering works. but I don't see this label in existing shoots/cloudprofile spec. Does it matter?

Reply:
This will only be created with Gardener v1.42, that's why it's disabled by default and must be enabled by setting `useObjectSelector=true` only when a high enough Gardener version is deployed.